### DISCUSSION TITLE
Welcome to pulsar-msg-filter-plugin Discussions!

### DISCUSSION THREAD

**Discussion by yangl** [yangl/pulsar-msg-filter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by praveengopalan** [comment]
Hi 
I need assistance configuring an entry filter in my local environment. I followed the instructions provided in the [README-en.md](https://github.com/yangl/pulsar-msg-filter/blob/main/README-en.md). However, after restarting the broker, the message filter is not functioning. I checked the broker log and did not see the message 'successfully loaded entry filter for name {name of your entry filter}'. Could you please help?
